.github/workflows/fprime-gds-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: fprime-gds tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
```

Comment:
We should actually test a few different versions of python to make sure the GDS works in most python environments. I think we should test against python 3.6, the oldest version of python we support, and python 3.9, the newest python release.

You can use github action build matricies to easily run the same CI workflow a few times with different versions of python